py/aou_workbench_client/swagger_client/models/table_query.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, table_name=None, columns=None, filters=None, order_by=None):
         self._order_by = None
         self.discriminator = None
 
-        self.table_name = table_name
+        self.table_name = table_name or (table.table_name if table else None)
```

Comment:
Clearer just to say `table.table_name if table else table_name`?

Reply:
Done.